mmedit/models/base_models/two_stage.py

```diff
@@ -213,7 +219,8 @@ def calculate_loss_with_type(self,
 
         return loss_dict
 
-    def train_step(self, data: List[dict], optim_wrapper):
+    def train_step(self, data: List[dict],
+                   optim_wrapper: Dict[str, torch.optim.Optimizer]) -> dict:
```

Comment:
`optim_wrapper` seems to be an`OptimWrapperDict`. I think you can add `from mmengine.optim import OptimWrapperDict` at the start of this file and then use `OptimWrapperDict` as typing hint.